rust/actyx/ax-types/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "ax_types"
+version = "0.1.0"
+authors = ["Actyx AG <developer@actyx.io>"]
+edition = "2021"
+description = "Actyx's core types"
+readme = "README.md"
+documentation = "https://docs.rs/actyx_sdk"
+homepage = "https://developer.actyx.com/"
+repository = "https://github.com/Actyx/Actyx"
+license = "Apache-2.0"
+keywords = ["distributed", "decentralized", "event-sourcing"]
+categories = ["network-programming"]
+
+[features]
+default = ["arb"]
+arb = ["quickcheck"]
+sqlite = ["rusqlite"]
```

Comment:
do we need this somewhere else than in the differential dataflow library? (actyxos_data_flow)

Reply:
Yes, `SqliteIndexStore` requires `ToSql` on `StreamId`